src/test/scala/com/gradle/internal/UtilsTest.scala

```diff
@@ -1,5 +1,3 @@
 package com.gradle.internal
 
-class UtilsTest {
-
-}
+class UtilsTest {}
```

Comment:
can we remove this?

Reply:
The plan is to add tests as we have in [Gradle](https://github.com/gradle/common-custom-user-data-gradle-plugin/blob/main/src/test/java/com/gradle/UtilsTest.java)/Maven, just haven't done that before the other things are finialized/